### PR TITLE
Add Login Page

### PR DESCRIPTION
**Pull request by jazwu**
## Description
**Login feature:** Provide username and password. If username doesn't exist or the password doesn't match, pop an error message. If successfully log in, then redirects to index page.
**Code modified:**
- add a login HTML
- add a login view

## How did you test the changes?
image
If username doesn't exist (e.g. type "aaaa" as username):
image
If username does exist, but the password doesn't match (e.g. type "sarthakgoel97" as username and type "123" as password): 
image
If username matches the password (e.g. username="sarthakgoel97", password="password"), then it redirects to the index page:
image

## Testing Screenshots
see above

### CODE REVIEW

File: healthScore/.gitignore
Comment (sarthakgoel1997) [inline healthScore/.gitignore:1]: Please avoid pushing the gitignore file to GitHub

**Review by sarthakgoel1997 — CHANGES_REQUESTED** [review]

**Comment by sarthakgoel1997** [comment]
Please take a pull of the current develop branch. I think the build is failing because of that

**Comment by jazwu** [comment]
> Please take a pull of the current develop branch. I think the build is failing because of that

coverage failed now, and we don't have test.py. I am confused about the reason. Do you have any ideas? @sarthakgoel1997

**Comment by jazwu** [comment]
> Please take a pull of the current develop branch. I think the build is failing because of that

I am confused about coveralls. I have updated to the latest develop branch and merged mine. Meanwhile, we don't have test.py. Do you have any ideas about how to pass coveralls? @sarthakgoel1997 Thank you!

**Comment by sarthakgoel1997** [comment]
Coveralls failed because the overall % dropped a bit. But I think that's okay, we can ignore it for now

File: healthScore/healthScore/urls.py
Comment (sarthakgoel1997) [inline healthScore/healthScore/urls.py:1]: A new file got created on the path: `healthScore/healthScore/urls.py`
Did you mean to add a new url path to `healthScore/urls.py` instead, as the file structure changed?

**Review by sarthakgoel1997 — CHANGES_REQUESTED** [review]

**Comment by sarthakgoel1997** [comment]
I think some binary files got pushed to the PR. Can you please remove them?

**Review by sarthakgoel1997 — APPROVED** [review]

**Pull request merged**